### ISSUE TITLE
count schematic ground parts as "ground fill seeds"

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#2510]
_From [irasc...@gmail.com](https://code.google.com/u/104729248032245122687/) on April 11, 2013 17:37:42_

What steps will reproduce the problem? 1. 2. 3. Attach your sketch file and/or custom part files to the bug report. If the sketch uses custom parts, save the sketch as shareable (under the file menu). What is the expected output? What do you see instead? What version of Fritzing are you using? On what operating system? Please use labels and text to provide additional information.

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=2512_

**Comment by failiz** [comment]
Still an issue. See below:
![image](https://github.com/fritzing/fritzing-app/assets/5708642/78b04e71-e688-4d92-b000-092bf9809087)
![image](https://github.com/fritzing/fritzing-app/assets/5708642/7584b40e-c781-45a2-9ca7-a3973ef73e5c)
The pad of the resistor is connected to a ground symbol in the schematic, but it is not connected to the ground plane in PCB. The sensor has a connector labeled "GND" and that connector is automatically connected to the ground plane.

**Comment by KjellMorgenstern** [comment]
The schematic ground currently does not have any corresponding element in the PCB. 
And the other way round, ground fills or copper fills don't have corresponding elements in the Schematic.
So, there is nothing we can connect, and we would have to add some data structure.

How about adding a symbol in the schematic for each ground fill? 
![image](https://github.com/fritzing/fritzing-app/assets/6494025/e782e2e6-60cb-4657-b327-d60b37cbe3db)
It doesn't need to look like a netlabel, maybe a different ground symbol would also work.

Existing sketches should not change. However, if a ground fill is added in future versions, there would automatically be a corresponding symbol in the schematic.

**Comment by failiz** [comment]
I do not think we should add a copper seed symbol in the schematic. I would just add all the connectors that are connected to ground symbols in the schematic to the seed list (we can search for them in the code). In the PCB options, users can remove them if they do not want them to be ground seeds. This would make the behavior more similar to the current one where all GND connectors of a part as added as ground seeds automatically and avoid having a "strange" non-schematic-related symbol in the SCH.

**Comment by KjellMorgenstern** [comment]
Ah yes. I looked at U1-7 in the schematic, because it is shown there as not connected.

But R1-1 in breadboard is also wrong, and that can indeed solved with probably not much difficulty by adding it to the ground fill seed list.
